src/legacy/core_plugins/timelion/server/series_functions/precision.js

```diff
@@ -31,14 +30,10 @@ export default new Chainable('precision', {
     {
       name: 'precision',
       types: ['number'],
-      help: i18n.translate('timelion.help.functions.precision.args.precisionHelpText', {
-        defaultMessage: 'Number of digits to round each value to',
-      }),
```

Comment:
You can keep the i18n ID, but you will need to search for the same id `timelion.help.functions.precision.args.precisionHelpText` in the translation files and delete the translation.

Reply:
@wylieconlon when you say `i18n ID`, do you mean lines 20, 34, and 39? Or just line 20?

Also, where are the translation files?